README.md

```diff
@@ -44,7 +44,14 @@
 
   ### Authorization
 
-  * [dry-rest-permissions](https://github.com/dbkaplan/dry-rest-permissions): Rules based permissions for the Django Rest Framework
+  * [dry-rest-permissions](https://github.com/dbkaplan/dry-rest-permissions): Rules based permissions.
+  * [drf-access-policy](https://github.com/rsinger86/drf-access-policy): Declarative access policies/permissions modeled after AWS' IAM policies.
+  * [djangorestframework-api-key](https://github.com/florimondmanca/djangorestframework-api-key): API key permissions.
+  * [djangorestframework-composed-permissions](https://github.com/niwinz/djangorestframework-composed-permissions): A simple way to define complex permissions.
```

Comment:
We should remove https://github.com/niwinz/djangorestframework-composed-permissions, https://github.com/caxap/rest_condition, https://github.com/computer-lab/django-rest-framework-roles, https://github.com/allisson/django-rest-framework-role-filters, https://github.com/drf-psq/drf-psq. They are not maintained anymore. 

However we can keep drf-access-policy and djangorestframework-api-key as they are maintained. Sounds good for you?

Reply:
Even if not maintained, they still work well to integrate with django-rest-framework. I think it would be nice to include them. What do you think?

Reply:
Seems like https://github.com/wsvincent/awesome-django also accepts unmaintained projects. Thank you for your contribution 👍

Reply:
Thanks @nioperas06